docs/sources/project/create-pr.md

```diff
@@ -56,28 +60,35 @@ Always rebase and squash your commits before making a pull request.
         pick 1a79f55 Tweak some of the other text for grammar
         pick 53e4983 Fix a link
         pick 3ce07bb Add a new line about RHEL
-        
-    If you run into trouble, `git --rebase abort` removes any changes and gets
-    you back to where you started. 
 
-4. Squash the `pick` keyword with `squash` on all but the first commit.
+    If you are not sure about the rebase at this point, delete all the lines and
+    quit from the editor which aborts the rebase process.
+        
+5. Replace the `pick` keyword with `squash` on all but the first commit.
 
         pick 1a79f55 Tweak some of the other text for grammar
         squash 53e4983 Fix a link
         squash 3ce07bb Add a new line about RHEL
 
-    After closing the file, `git` opens your editor again to edit the commit
-    message. 
+    Save the changes and quit from editor at which point, `git` will start the actual 
```

Comment:
This is actually a step and the "at which point" is the system response or effect of the step.  I'm not sure that we need to go into a full explanation of how to resolve a conflict or abort a rebase.  Again, it introduces uncertainty and `git` provides help for resolving any conflict the user encounters.

---> instead

6. Save and quit the editor.

  After you quit, `git` starts the rebase. As it rebases, `git` reports it progress.  Sometimes your changes can conflict with the work of others.  If `git` encounters a conflict, it stops the rebase, and prints guidance for correcting the conflict.